sdk/redisenterprise/Azure.ResourceManager.RedisEnterprise/CHANGELOG.md

```diff
@@ -1,14 +1,14 @@
 # Release History
 
-## 1.2.0-beta.3 (Unreleased)
+## 1.2.0-beta.3 (2025-05-01)
 
 ### Features Added
 
 ### Breaking Changes
```

Comment:
I think your change is not breaking - therefore these entries should be under the `Features Added` section.
And we do not really have breaking changes in this change, therefore we could remove this line of `breaking changes`